### PR TITLE
Change instafyle content

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
     <form #formContainer [formGroup]="fg" class="add-edit-expense--form">
       <div>
         <div *ngIf="instaFyleCancelled">
-          Instafyle is taking longer than expected due to a bad network, please add the details manually
+          Extracting receipt details is taking longer than expected due to a bad network, please add the details manually
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.html:65]: Old: Image from iOS

File: src/app/fyle/my-expenses/add-expense-popover/add-expense-popover.component.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     </div>
     <div class="add-expense-popover-internal--content">
       <div class="add-expense-popover-internal--head">
-        Instafyle
+        Capture Receipt
```
Comment (Dimple16) [inline src/app/fyle/my-expenses/add-expense-popover/add-expense-popover.component.html:8]: Screenshot 2021-06-24 at 10 37 25 PM

File: src/app/fyle/my-profile/my-profile.page.html [hunk 3]
```diff
@@ -78,7 +78,7 @@
     <!-- Instafyle card start -->
     <div class="my-profile--card my-profile--card__action" *ngIf="orgUserSettings?.insta_fyle_settings?.allowed">
       <div class="my-profile--card-title">
-        Instafyle
+        Extract receipt details
```
Comment (Dimple16) [inline src/app/fyle/my-profile/my-profile.page.html:81]: Screenshot 2021-06-24 at 10 38 13 PM

File: src/app/fyle/my-expenses/add-expense-popover/add-expense-popover.component.html (hunk 2 above)
Comment (abhishek1234321) [reply src/app/fyle/my-expenses/add-expense-popover/add-expense-popover.component.html:8]: let us remove the `new` part

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**